tests/test_domain_json.py

```diff
@@ -0,0 +1,59 @@
+# work around until folder structure is updated
+import os
+import sys
+# insert root directory into python module search path
+sys.path.insert(1, os.getcwd())
+
+from io import StringIO
+import json
+from pathlib import Path
+import pytest
+
+from openmethane_prior.omInputs import domainPath
+from scripts.omDomainJSON import write_domain_json
+
+ROOT_DIRECTORY = Path(__file__).parent.parent
+
+@pytest.mark.skipif(os.path.isfile(domainPath) != True,
+                    reason="test requires omCreateDomainInfo.py to have been run")
```

Comment:
This unfortunately means the test will not run in CI.

Due to how `omInputs.py` and `omCreateDomainInfo.py` work, I'm not sure how this could be handled better. If the domain file doesn't exist at the time of **importing**  `omInputs.py` , the tests will fail.

Running `omCreateDomainInfo.py` in the scope of the test doesn't even help, because it doesn't re-create the exported properties from `omInputs`. :shrug:

Reply:
Can you make an issue? That is a flaw in how the module is structured. Ideally importing modules should have no side effects. Easy fix

Reply:
Raised in issue https://github.com/openmethane/openmethane-prior/issues/23.